homeassistant/scripts/check_config.py

```diff
@@ -345,14 +345,21 @@ def _comp_error(ex, domain, config):
                 _comp_error(ex, domain, config)
                 continue
 
-        if not hasattr(component, 'PLATFORM_SCHEMA'):
+        if (not hasattr(component, 'PLATFORM_SCHEMA') and
+                not hasattr(component, 'PLATFORM_SCHEMA_BASE')):
             continue
 
         platforms = []
         for p_name, p_config in config_per_platform(config, domain):
             # Validate component specific platform schema
             try:
-                p_validated = component.PLATFORM_SCHEMA(p_config)
+                if hasattr(component, 'PLATFORM_SCHEMA_BASE'):
```

Comment:
Should platform_schema not have a higher priority?
Never used platform_schema_base, not sure what the rules are

Reply:
No, PLATFORM_SCHEMA_BASE should have higher priority.
Background here: #20224